### ISSUE TITLE
use Exception chaining with Java 3 Exception

### ISSUE THREAD

**Issue by activeeon-bot** [ow2-proactive/programming#266]
<a href="https://jira.activeeon.com/browse/PROACTIVE-376" title="PROACTIVE-376">Original issue</a> created by <a href="mailto:cedric.dalmasso_AT_activeeon.com">Cédric Dalmasso</a> on 20, Jun 2008 at 11:32 AM - PROACTIVE-376

<hr />

Exception defined in the Fractal API are Java 3 compliant and do not define a chaining exception constructor. Anyway, we can chain exception using the initCause method.

Basically replace
        } catch (NodeException e) {
            throw new InstantiationException(e.getMessage());
        }
by
        } catch (NodeException e) {
            InstantiationException ie = new InstantiationException(e.getMessage());
            ie.initCause(e);
            throw ie;
        }
